pkg/cmd/services_test.go

```diff
@@ -18,6 +20,170 @@ import (
+			},
+			ExpectError: true,
+			ValidateErr: func(t *testing.T, err error) {
+				if err == nil {
+					t.Fatalf("expected an error but did not get one")
+				}
+				if err.Error() != "expected a serviceInstanceID" {
+					t.Fatalf("expected error to be %s but got %v", "expected a serviceInstanceID", err)
+				}
+			},
+			Flags: []string{"--namespace=test", "-o=json"},
+			Args:  []string{},
+		},
+
+		{
+			Name: "test if error occurs getting service instance that an error is returned",
+			SvcCatalogClient: func() versioned.Interface {
+				fake := &scFake.Clientset{}
+				fake.AddReactor("get", "serviceinstances", func(action ktesting.Action) (handled bool, ret runtime.Object, err error) {
+					return true, nil, fmt.Errorf("error in get")
+				})
+				return fake
+			},
+			K8Client: func() kubernetes.Interface {
+				return &kFake.Clientset{}
+			},
+			ExpectError: true,
+			ValidateErr: func(t *testing.T, err error) {
+				if err == nil {
+					t.Fatalf("expected an error but did not get one")
+				}
+				if err.Error() != "error in get" {
+					t.Fatalf("expected error to be %s but got %v", "error in get", err)
+				}
+			},
+			Flags: []string{"--namespace=test", "-o=json"},
+			Args:  []string{"someid"},
+		},
+		{
+			Name: "test if error occurs deleting service instance that an error is returned",
+			SvcCatalogClient: func() versioned.Interface {
+				fake := &scFake.Clientset{}
+				fake.AddReactor("get", "serviceinstances", func(action ktesting.Action) (handled bool, ret runtime.Object, err error) {
+					return true, nil, nil
+				})
+				fake.AddReactor("delete", "serviceinstances", func(action ktesting.Action) (handled bool, ret runtime.Object, err error) {
+					return true, nil, fmt.Errorf("error in delete")
+				})
+				return fake
+			},
+			K8Client: func() kubernetes.Interface {
+				return &kFake.Clientset{}
+			},
+			ExpectError: true,
+			ValidateErr: func(t *testing.T, err error) {
+				if err == nil {
+					t.Fatalf("expected an error but did not get one")
+				}
+				if err.Error() != "error in delete" {
+					t.Fatalf("expected error to be %s but got %v", "error in delete", err)
+				}
+			},
+			Flags: []string{"--namespace=test", "-o=json"},
+			Args:  []string{"someid"},
+		},
+		{
+			Name: "test successful delete",
+			SvcCatalogClient: func() versioned.Interface {
+				fake := &scFake.Clientset{}
+				fake.AddReactor("get", "serviceinstances", func(action ktesting.Action) (handled bool, ret runtime.Object, err error) {
+					return true, &v1beta1.ServiceInstance{
+						ObjectMeta: metav1.ObjectMeta{GenerateName: "test"},
+					}, nil
+				})
+				fake.AddReactor("delete", "serviceinstances", func(action ktesting.Action) (handled bool, ret runtime.Object, err error) {
+					return true, nil, nil
+				})
+				return fake
+			},
+			K8Client: func() kubernetes.Interface {
+				return &kFake.Clientset{}
+			},
+			ExpectError: false,
+			ValidateErr: func(t *testing.T, err error) {
```

Comment:
I see a defined ```ValidateOutput``` in the struct but not seeing it used here. Is there any none error output to validate?